packages/web/src/pages/search-page-v2/SearchPageV2.tsx

```diff
@@ -0,0 +1,178 @@
+
+type SearchHeaderProps = {
+  category?: CategoryKey
+  setCategory: (category: CategoryKey) => void
+  title: string
+  query: Maybe<string>
+}
+
+const SearchHeader = (props: SearchHeaderProps) => {
+  const { category: categoryKey = 'all', setCategory, query, title } = props
+
+  const { isMobile } = useMedia()
+
+  const handleChange = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      const value = e.target.value
+      setCategory(value as CategoryKey)
+    },
+    [setCategory]
+  )
+
+  const CategoryRadioGroup = (
+    <RadioGroup
+      direction='row'
+      gap='s'
+      aria-label={'Select search category'}
+      name='searchcategory'
+      value={categoryKey}
+      onChange={handleChange}
+    >
+      {Object.entries(categories)
+        .filter(([key]) => !isMobile || key !== 'all')
+        .map(([key, category]) => (
+          <SelectablePill
+            aria-label={`${key} search category`}
+            icon={(category as Category).icon}
+            key={key}
+            label={capitalize(key)}
+            size='large'
+            type='radio'
+            value={key}
+            checked={key === categoryKey}
+          />
+        ))}
+    </RadioGroup>
+  )
+
+  return isMobile ? (
+    <Flex p='s' css={{ overflow: 'scroll' }}>
+      {CategoryRadioGroup}
+    </Flex>
+  ) : (
+    <Header
+      {...props}
+      primary={title}
+      secondary={
+        query ? (
+          <Flex ml='l'>
+            <Text variant='heading' strength='weak'>
+              &#8220;{query}&#8221;
+            </Text>
+          </Flex>
+        ) : null
+      }
+      rightDecorator={CategoryRadioGroup}
+      variant='main'
+    />
+  )
+}
+
+export const SearchPageV2 = () => {
+  const { isMobile } = useMedia()
+
+  const { category, query } = useParams<{
+    category: CategoryKey
+    query: string
+  }>()
+  const { history } = useHistoryContext()
+
+  // Set nav header
+  const { setLeft, setCenter, setRight } = useContext(NavContext)!
+  useEffect(() => {
+    setLeft(LeftPreset.BACK)
+    setCenter(CenterPreset.LOGO)
+    setRight(RightPreset.SEARCH)
+  }, [setLeft, setCenter, setRight])
+
+  const setCategory = useCallback(
+    (category: CategoryKey) => {
+      history.push(`/search/${query}/${category}`)
+    },
+    [history, query]
+  )
+
+  const header = (
+    <SearchHeader
+      query={query}
+      title={'Search'}
+      category={category as CategoryKey}
+      setCategory={setCategory}
+    />
+  )
+
+  const PageComponent = isMobile ? MobilePageContainer : Page
+
+  return (
+    <PageComponent
+      title={`${query}`}
+      description={`Search results for ${query}`}
+      // canonicalUrl={fullSearchResultsPage(query)}
+      header={header}
+    >
+      <Flex direction='column' css={{ width: '100%' }}>
+        {isMobile ? header : null}
+        {!query ? <SearchCatalogTile /> : null}
+        {status === Status.LOADING ? <LoadingSpinner /> : <div></div>}
```

Comment:
`<div/>`

Reply:
This is where the search results will go, just a placeholder for now!